openapi/components/schemas/Company.yaml

```
@@ -1,33 +1,39 @@
 type: object
 description: |-
-  Company information associated with the customer primary email address domain.
+  Company information that is associated with the customer's primary email address domain.
   
   This is a paid feature, to enable it [contact Rebilly](https://www.rebilly.com/support/).
 nullable: true
 readOnly: true
 properties:
   name:
     type: string
+    description: Name of the company.
     nullable: true
   domain:
     type: string
     nullable: true
   yearFounded:
     type: number
+    description: Year the company was founded.
```

suggestion: fixes the lint issue
```suggestion
    type: number
    description: Founding year of the company.
```